lib/elixir/lib/calendar/time.ex

```diff
@@ -321,6 +321,43 @@ defmodule Time do
   end
 
   @doc """
+  Adds the `number` of `unit`s to the given `time`.
+
+  This function accepts the `number` measured according to `Calendar.ISO`.
+  The time is returned in the same calendar as it was given in.
+
+  Note the result value represents time of day,
+  meaning that it is cyclic and will never go over 24 hours.
+
+  ## Examples
+
+      iex> Time.add(~T[10:00:00], 27000)
+      {:ok, ~T[17:30:00.000000]}
+      iex> Time.add(~T[11:00:00.005], 2400)
+      {:ok, ~T[11:40:00.005000]}
+      iex> Time.add(~T[00:00:00], 86399999, :millisecond)
+      {:ok, ~T[23:59:59.999000]}
+      iex> Time.add(~T[17:10:05], 86400)
+      {:ok, ~T[17:10:05.000000]}
+      iex> Time.add(~T[23:00:00], -60)
+      {:ok, ~T[22:59:00.000000]}
+
+  """
+  @spec add(Calendar.time, integer, System.time_unit) :: t
```

Comment:
Doesn't this return `{:ok, t} | ...`?

Reply:
Oops, it is the opposite: need to change the return value.